logger/mmap_plugin.cc

```diff
@@ -0,0 +1,166 @@
+/**
```

Comment:
Do we need to separate this from mmap_processor.cc? Having that we're going to have quite a few plugins and all of them are pretty small, I'd prefer to keep the source tree smaller